src/Compilers/Core/Portable/InternalUtilities/SpecializedCollections.cs

```diff
@@ -11,7 +11,7 @@ internal static partial class SpecializedCollections
 
         public static T[] EmptyArray<T>()
```

Comment:
Yes, we can. There is more cleanup that can be done. I was trying to limit the change to minimum